test/e2e/helpers_test.go

```diff
@@ -47,6 +47,9 @@ var (
 
 	//go:embed resources/config/rancher-setting-patch.yaml
 	rancherSettingPatch []byte
+
+	//go:embed resources/config/nginx-ingress.yaml
```

Comment:
It is better to have nginx installed with helm here, in case we’d need to use different image or version in the future. Plus waiting for rollout could be delegated to helm in `setupIngress`

Reply:
I like that these manifests are provided as an example with proper ports, services, etc. configured to work with a kind cluster. It's a no-brainer for a test environment. We are very unlikely to change it in the future. Kind docs also list it at a recommended way to install.

Reply:
Though you still base the connectivity setup on the existing ingress configuration for the rancher pod, where the example suggests to open extra port mappings you don’t (can’t) add, so what we have here is not a  recommended way. Because of that this is not guarantied that this configuration won’t change in the future if we change ingress config.